client/src/style/App.css

```diff
@@ -0,0 +1,10 @@
+body{
+    background-color: white 
+
```

Comment:
Is this blank lines need?

Reply:
yes, its start to modify body,  in class we must speak about design main page.

Reply:
I mean line 3 and 4. Is it need?

Reply:
empty line?

Reply:
yes

Reply:
no

Reply:
+